app/controllers/admin/pdfs_controller.rb

```diff
@@ -14,6 +14,6 @@ def create
   end
 
   def pdf_params
-    params.require(:pdf).permit(:pdf_file, :url, :page, :slot)
+    params.require(:pdf).permit(:pdf_file, :url, :slot)
```

Comment:
Are we mixing pdf and video attributes here?  I thought a video has a url but a pdf does not.  If a pdf should have a url, I don't see the url in the schema.